Sources/Core/UI/Viewers/PDF/PDFViewController.swift

```diff
@@ -535,8 +535,23 @@ extension PDFViewController: SearchViewControllerDelegate {
 
 extension PDFViewController: PDFViewDelegate {
     public func pdfViewWillClick(onLink _: PDFView, with url: URL) {
-        let safaryVC = SFSafariViewController(url: url)
-        present(safaryVC, animated: true, completion: nil)
+        var URLCopy = url
```

Comment:
Hey Sujay,
Thanks a lot for fixing a PDF crash! We really appreciate new Box SDK. 
What do you think about the following suggestion?
```
        if let finalURL = url.scheme == nil ? URL(string: "https://" + url.absoluteString) : url, ["http", "https"].contains(finalURL.scheme!) {
            let safariVC = SFSafariViewController(url: finalURL)
            present(safariVC, animated: true, completion: nil)
        } else {
            UIApplication.shared.open(url) { (result) in
                if !result {
                    DispatchQueue.main.async {
                        self.showAlertWith(title: "Error", message: url.absoluteString + " is an invalid link that can't be handled by any application.")
                    }
                }
            }
        }
```
Best wishes,
LZ team